package.json

```diff
@@ -34,8 +34,10 @@
     "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
     "lint": "repo-tools lint --cmd eslint -- build/src/ samples/ build/system-test/ build/test/",
     "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js test/*/*/*.js system-test/*.js system-test/*/*.js",
+    "presystem-test": "npm run compile",
```

Comment:
Does the `pre` prefix work for any arbitrary scripts?

Reply:
Yes.  I've tested it locally.

Reply:
All right. Good to know.